dockstore-common/src/main/java/io/dockstore/common/HttpStatusMessageConstants.java

```diff
@@ -0,0 +1,20 @@
+package io.dockstore.common;
+
+/**
+ * Constants for Http Status code descriptions.
+ */
+public final class HttpStatusMessageConstants {
+    public static final String OK = "OK";
```

Comment:
Try some https://stackoverflow.com/questions/1637750/java-library-to-map-http-status-code-to-description and see if any of them works

Reply:
Tried them and they don't work because the annotation description requires that the `Attribute value must be constant`. Using a method call to get the description makes it not a constant